### DISCUSSION TITLE
Why is there a `#` at the start of `consult-ripgrep` by default?

### DISCUSSION THREAD

**Discussion by NightMachinery** [minad/consult#949] [Q&A] [answered]
Why is there a `#` at the start of `consult-ripgrep` by default?
<img width="834" alt="image" src="https://github.com/minad/consult/assets/36224762/9aa1eb34-00bd-48cb-b6d9-a458ac832c7a">

Deleting it also doesn't seem to do anything.

**Comment by minad** [comment]
Please read the documentation. Thanks.

**Reply by NightMachinery** [reply, marked as answer]
https://github.com/minad/consult#grep-and-find
